docs/examples/globaleventshandlerprovider/sheetMobileHandlers.js

```diff
@@ -0,0 +1,56 @@
+// @flow strict
+import { type Node, useState } from 'react';
+import {
+  Flex,
+  Layer,
+  SheetMobile,
+  Box,
+  DeviceTypeProvider,
+  Button,
+  FixedZIndex,
+  CompositeZIndex,
+  GlobalEventsHandlerProvider,
+} from 'gestalt';
+
+export default function Example(): Node {
+  const [showComponent, setShowComponent] = useState<null | boolean>(true);
+
+  const PAGE_HEADER_ZINDEX: FixedZIndex = new FixedZIndex(10);
+  const ABOVE_PAGE_HEADER_ZINDEX: CompositeZIndex = new CompositeZIndex([PAGE_HEADER_ZINDEX]);
+
+  return (
+    <GlobalEventsHandlerProvider
+      sheetMobile={{
```

Comment:
Should we demonstrate proper Provider value best practices in the examples, by memoizing the passed-down objects and contained callbacks?
```js
const sheetMobileOnOpen = useCallback(() => console.log('On open handler'), []);
const sheetMobileOnClose = useCallback(() => console.log('On close handler'), []);
const sheetMobileHandlers = useMemo(() => ({ onOpen: sheetMobileOnOpen, onClose: sheetMobileOnClose }), []);

...

<GlobalEventsHandlerProvider sheetMobile={sheetMobileHandlers}>
```
const sheetMobileHandlers =